src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/FindPal.cs

```diff
@@ -360,7 +360,7 @@ private static bool VerifyCertificateIgnoringErrors(SafeCertContextHandle pCertC
             // This needs to be kept in sync with IsCertValid in the
             // Unix/OpenSSL PAL version (and potentially any other PALs that come about)
             ChainPal chainPal = ChainPal.BuildChain(
-                true,
+                false,
```

Comment:
Nit: it'd be nice if this were named, e.g. ```useMachineContext: false```... I had to go look up what this first argument was.